examples/pytorch/diffpool/README.md

```diff
@@ -0,0 +1,28 @@
+Hierarchical Graph Representation Learning with Differentiable Pooling
+============
+
+
+Paper link: [https://arxiv.org/abs/1806.08804](https://arxiv.org/abs/1806.08804)
+
+Author's code repo: [https://github.com/RexYing/diffpool](https://github.com/RexYing/diffpool)
+
+This folder contains a DGL implementation of the DiffPool model. The first pooling layer is computed with DGL, and following pooling layers are computed with tensorized operation since the pooled graphs are dense.
+
+Dependencies
+------------
+* PyTorch 1.0+
+
+How to run
+----------
+
+```bash
+python train.py --dataset ENZYMES --pool_ratio 0.10 --linkpred True --num_pool 1
```

Comment:
Also add sample code to run DD. And describe the corresponding network structure for both dataset.